packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline-results.modal.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { ModalHeader, ModalBody } from '@carbon/react';
+import Trendline from '../trendline/trendline.component';
+import { basePath } from '@openmrs/esm-patient-chart-app/src/constants';
```

Comment:
We shouldn't do this, right, @ibacher?

Reply:
What is recommended?

Reply:
Right... I'd also say that we make absolutely no guarantee to ensure that importing from apps like this will work. In fact, it _should_ be expected to fail at runtime (although this kind of narrow case works).

Anything that needs to be shared across apps in the chart belongs in the patient-common-lib. That's the whole reason that exists.

Reply:
> We shouldn't do this, right, @ibacher?

Addressed this.